week-02/assessment/gomoku/src/main/java/learn/gomoku/UiGomoku.java

```diff
@@ -0,0 +1,136 @@
+package learn.gomoku;
+import learn.gomoku.game.Gomoku;
+import learn.gomoku.game.Result;
+import learn.gomoku.game.Stone;
+import learn.gomoku.players.HumanPlayer;
+import learn.gomoku.players.Player;
+import learn.gomoku.players.RandomPlayer;
+
+
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class UiGomoku {
+
+    // Welcome message method
+    public static void welcomeMessage() {
+        System.out.println("Welcome to Gomoku");
+        System.out.println(("=").repeat(17));
+
+    }
```

Comment:
The static modifier should be used very sparingly, please remove from all of your methods.
```suggestion
    public void welcomeMessage() {
        System.out.println("Welcome to Gomoku");
        System.out.println(("=").repeat(17));

    }
```